src/alchemlyb/estimators/ti_.py

```diff
@@ -105,3 +105,50 @@ def fit(self, dHdl):
         self.states_ = means.index.values.tolist()
 
         return self
+
+    def separate_dhdl(self):
+        """
+        For transitions with multiple lambda, the `dhdl` attribute would return
+        a DataFrame which gives the dHdl for all the lambda states, regardless
+        of whether it is perturbed or not. This function creates a list of
+        Series for each lambda, where each Series describes the potential energy
+        gradient for the lambdas state that is perturbed.
+
```

Comment:
You can use intersphinx links such as
```reST
.... :class:`~pandas.DataFrame` ... :class:`pandas.Series` ...
```
see the [python roles in sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects)